girder_worker/app.py

```diff
@@ -130,10 +132,30 @@ def call_item_task(self, inputs, outputs={}):
         return self.run.call_item_task(inputs, outputs)
 
 
-@before_task_publish.connect
+@before_task_publish.connect # noqa
```

Comment:
Why `# noqa`? Too complex?

Reply:
Unfortunately yes. I am planning to reduce the complexity of that function in another PR.